latch/types/directory.py

```diff
@@ -35,13 +35,13 @@ def task(dir: LatchDir):
     latch or s3 url, can be inspected from an object passed to the task to
     reveal its remote source.
 
-    It can also to deposit the file to a latch path when the directory is
+    It can also to deposit the directory to a latch path when the directory is
```

Comment:
these are docstring change I figured I'd make along the way

Reply:
ty